contracts/src/rollup/IRollupCore.sol

```diff
@@ -10,6 +10,7 @@ import "../bridge/IOutbox.sol";
 import "../bridge/IInboxBase.sol";
 import "./IRollupEventInbox.sol";
 import "../challengeV2/EdgeChallengeManager.sol";
+import "../challengeV2/IAssertionChain.sol";
```

Comment:
i added this because i was having compiler issues with `IAssertionChain` being undefined

Reply:
this is causing a circular dependency I believe, let's have the whitelist functions in `IAssertionChain` and avoid importing `IRollupCore` in `EdgeChallengeManager`